app/src/backend/environmentVariables/hooks/useEnvironmentVariables.ts

```diff
@@ -85,13 +78,26 @@ const useEnvironmentVariables = () => {
     return removeEnvironmentVariable(ownerId, { environment, key });
   };
 
+  const renderVariables = (template: string) => {
```

Comment:
Lets rename this to `renderString` and create another method `render` which can render JSON object while recursively going through every keys.